### PR TITLE
docs: improvement-docs

### PR DESCRIPTION
**Pull request by haroon786**

### CODE REVIEW

File: projects/bulma-app/src/assets/docs/hero/header-description.md [hunk 1]
```diff
@@ -4,7 +4,25 @@
 
 The hero component allows you to add a **full width banner** to your webpage, which can optionally cover the full height of the page as well.
 
-The basic requirement of this component are:
+To display a Hero component in your app and verify that everything works.
+You need to import the BulmaHeroModule that you want to display by adding the following lines to your app.module.ts file.
```
Comment (AnkitSharma-007) [inline projects/bulma-app/src/assets/docs/hero/header-description.md:8]: Add inline code syntax for BulmaHeroModule and app.module.ts

File: projects/bulma-app/src/assets/docs/navbar/header-description.md [hunk 2]
```diff
@@ -2,13 +2,29 @@
...
-    - `navbar-burger` the hamburger icon, which toggles the navbar menu on touch devices
-  - `navbar-menu` the right side, hidden on touch devices, visible on desktop
-    - `navbar-start` the left part of the menu, which appears next to the navbar brand on desktop
-    - `navbar-end` the right part of the menu, which appears at the end of the navbar
-      - `navbar-item` each single item of the navbar, which can either be an a or a div
-        - `navbar-link` a link as the sibling of a dropdown, with an arrow
-        - `navbar-dropdown` the dropdown menu, which can include navbar items and dividers
-          - `navbar-divider` a horizontal line to separate navbar items
+To display a Navbar component in your app and verify that everything works.
+You need to import the BulmaNavbarModule that you want to display by adding the following lines to your app.module.ts file.
```
Comment (AnkitSharma-007) [inline projects/bulma-app/src/assets/docs/navbar/header-description.md:6]: Add inline code syntax for BulmaNavbarModule and app.module.ts

File: projects/bulma-app/src/assets/docs/panel/panel-text.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
-# Panel 
-##### To emphasize important notes in your application ,The utilitarian ```panel```  simple  to use.
+# Panel
+
+##### To emphasize important notes in your application ,The utilitarian `panel` simple to use.
+
 To display a panel component in your app and verify that everything works. You need to import the BulmaPanelModule that you want to display by adding the following lines to your app.module.ts file.
```
Comment (AnkitSharma-007) [inline projects/bulma-app/src/assets/docs/panel/panel-text.md:5]: Add inline code syntax for BulmaPanelModule and app.module.ts

**Comment by AnkitSharma-007** [comment]
LGTM

File: projects/bulma-app/src/assets/docs/progressbar/progressbar-text.md [hunk 4]
```diff
@@ -16,4 +16,10 @@ import { BulmaProgressbarModule} from 'ngx-bulma'
 ### Basic
 
 To use the Progressbar component in your Angular application.
-Add `buTextarea` in textarea tag.
+Add `buProgress` in progress tag.
+
+| Property     | Description                                   |     |     |     |
```
Comment (santoshyadav198613) [inline projects/bulma-app/src/assets/docs/progressbar/progressbar-text.md:21]: Properties

File: projects/bulma-app/src/assets/docs/navbar/header-description.md [hunk 5]
```diff
@@ -2,13 +2,29 @@
...
+| Element             | Description                                                                        |     |     |     |
+| ------------------- | ---------------------------------------------------------------------------------- | --- | --- | --- |
+| `<bu-nav>`          | Parent component                                                                   |     |     |     |
+| `<bu-nav-brand>`    | the left side **always visible.**, which usually contains the logo and optionally. |
+| `<bu-navbar-start>` | the left part of the menu, which appears next to the navbar brand on desktop.      |
+| `<bu-navbar-end>`   | the right part of the menu, which appears at the end of the navbar.                |
+
+| Propert           | Description                                                        |     |     |     |
+| ----------------- | ------------------------------------------------------------------ | --- | --- | --- |
+| `navbar-burger`   | the hamburger icon, which toggles the navbar menu on touch devices |     |     |
```
Comment (santoshyadav198613) [inline projects/bulma-app/src/assets/docs/navbar/header-description.md:26]: we don't have any such properties

File: projects/bulma-app/src/assets/docs/input/input-text.md [hunk 6]
```diff
@@ -2,19 +2,33 @@
...
 ```
+
+### Basic
+
+To use the Input component in your Angular application.
+Add `buInput` in Input tags.
+
+| Property  | Description                                    |     |     |     |
+| --------- | ---------------------------------------------- | --- | --- | --- |
+| `buInput` | For initializing buInput in your HTML          |     |     |     |
```
Comment (santoshyadav198613) [inline projects/bulma-app/src/assets/docs/input/input-text.md:29]: This is directive.

File: projects/bulma-app/src/assets/docs/hero/header-description.md [hunk 7]
```diff
@@ -4,7 +4,25 @@
...
+
+### Basic
+
+To use the Container component in your Angular application.
+Add `buContainer` in any div tags.
+
+| Property      | Description                                       |     |     |     |
+| ------------- | ------------------------------------------------- | --- | --- | --- |
+| `bu-hero`     | Parent Hero Component.                            |     |     |     |
+| `buContainer` | For initializing Container in your HTML.          |
```
Comment (santoshyadav198613) [inline projects/bulma-app/src/assets/docs/hero/header-description.md:25]: This directive is not related to hero

File: projects/bulma-app/src/assets/docs/textarea/textarea-text.md [hunk 8]
```diff
@@ -12,4 +12,13 @@ import { BulmaTextareaModule} from 'ngx-bulma'
  });
 ```
 
-#### Use the Textarea component in your Angular application ,add `buTextarea` in progress tag.
+TO Use the Textarea component in your Angular application ,add `buTextarea` in textarea tag.
+
+| Element      | Description                                       |     |     |     |
+| ------------ | ------------------------------------------------- | --- | --- | --- |
+| `buTextarea` | For initializing Textarea in your HTML.           |     |     |     |
+| `theme`      | For specifying different themes to Input          |     |     |     |
```
Comment (santoshyadav198613) [inline projects/bulma-app/src/assets/docs/textarea/textarea-text.md:20]: This is a property

File: projects/bulma-app/src/assets/docs/navbar/header-description.md [hunk 9]
```diff
@@ -2,13 +2,29 @@
...
+The most basic Modal needs only of `<bu-nav>` element with some content. However, Bulma components provides a number of preset sections that you can use inside of `</bu-nav>`
+
+| Element             | Description                                                                        |     |     |     |
+| ------------------- | ---------------------------------------------------------------------------------- | --- | --- | --- |
+| `<bu-nav>`          | Parent component                                                                   |     |     |     |
+| `<bu-nav-brand>`    | the left side **always visible.**, which usually contains the logo and optionally. |
+| `<bu-navbar-start>` | the left part of the menu, which appears next to the navbar brand on desktop.      |
+| `<bu-navbar-end>`   | the right part of the menu, which appears at the end of the navbar.                |
+
+| Propert           | Description                                                        |     |     |     |
```
Comment (AnkitSharma-007) [inline projects/bulma-app/src/assets/docs/navbar/header-description.md]: The spelling is wrong. It should be "Properties" instead of "Propert"

File: projects/bulma-app/src/assets/docs/input/input-text.md [hunk 10]
```diff
@@ -2,19 +2,36 @@
...
 ```
+
+### Basic
+
+To use the Input component in your Angular application.
+Add `buInput` in Input tags.
+
+| Directive | Description                           |     |     |     |
+| --------- | ------------------------------------- | --- | --- | --- |
+| `buInput` | For initializing buInput in your HTML |     |     |     |
```
Comment (AnkitSharma-007) [inline projects/bulma-app/src/assets/docs/input/input-text.md:29]: I am not sure why we have 3 extra columns in this table. I have seen extra columns in all the tables in this PR. @santoshyadav198613 - Do we need to have extra columns?

File: projects/bulma-app/src/assets/docs/navbar/header-description.md (hunk 9 above)
Comment (haroon786) [reply projects/bulma-app/src/assets/docs/navbar/header-description.md]: this  one  already changed, why it is showing here 
des
